### PR TITLE
Add kafka-dropwizard-reporter

### PR DESCRIPTION
**Pull request by jklukas**
Add a reference to https://github.com/SimpleFinance/kafka-dropwizard-reporter

### CODE REVIEW

File: _data/modules/thirdparty.yaml [hunk 1]
```diff
@@ -89,6 +89,14 @@
   license: MIT
   maven:
     url: http://mvnrepository.com/artifact/ru.vyarus/dropwizard-guicey
+- name: kafka-dropwizard-reporter
+  url: https://github.com/SimpleFinance/kafka-dropwizard-reporter
+  description: Register built-in Kafka client metrics to Dropwizard Metrics
+  dropwizard: 1.0.0, 0.7+
```
Comment (jklukas) [inline _data/modules/thirdparty.yaml:95]: I'm not quite sure how to express version compatibility here. This module only requires dropwizard-metrics 3.0+, so is compatible with dropwizard 0.7 to the latest. Is there a preferred pattern?

**Comment by joschi** [comment]
@jklukas This project is for listing Dropwizard addons but the project you're trying to add is a [Dropwizard Metrics](https://github.com/dropwizard/metrics) reporter.

You might want to add it to the docs of the Metrics project at https://github.com/dropwizard/metrics/tree/master/docs

**Comment by jklukas** [comment]
Thanks for suggesting a better home for this. Will pursue.

**Comment by joschi** [comment]
@jklukas Thank you for sharing this with us. 😄

**Pull request closed without merging**